keystores/src/main/java/com/palantir/conjure/java/config/ssl/KeyStores.java

```diff
@@ -112,14 +115,16 @@ static KeyStore createTrustStoreFromCertificates(Map<String, PemX509Certificate>
         keyStore = createKeyStore();
 
         for (Map.Entry<String, PemX509Certificate> entry : certificatesByAlias.entrySet()) {
-            try (InputStream certIn = new ByteArrayInputStream(
-                    entry.getValue().pemCertificate().getBytes(StandardCharsets.UTF_8))) {
+            try (
```

Comment:
@dansanduleac any idea if we can make try-with-resources not _always_ have a newline here?

Reply:
Yep! https://github.com/palantir/gradle-baseline/pull/835